### PR TITLE
Fixed issue #1 - load images asynchronously

### PR DESCRIPTION
**Pull request by gornostal**
#1

### CODE REVIEW

File: ulauncher/results/IconLoader.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    Loads icons asynchronously
+
+    Usage:
+        iconLoader = IconLoader.load(src, lambda pixbuf: set_from_pixbuf(pixbuf))
+        if not iconLoader.is_loaded:
+            set_from_pixbuf(iconLoader.icon_loading)
+
+    """
+
+    # public
```
Comment (yatsenko) [inline ulauncher/results/IconLoader.py]: Why did you add such comments as "public" and "private" properties and didn't use native python way of define private properties ( __private_property ).
Anyway, it's not good practice to use public properties, better to use private properties + setters, getters. Does it make sence?

**Comment by gornostal** [comment]
@yatsenko PR is fixed. Please review.
